lib/worker.js

```diff
@@ -79,9 +79,9 @@ class AssetComputeWorker {
             this.actionTimeoutId = this.sendEventsBeforeActionTimeout();
         }
 
-        if (options.postProcessingEngine) {
-            this.postProcessingEngine = options.postProcessingEngine;
-        }
+        // if (options.postProcessingEngine) {
```

Comment:
Just remove it?

Reply:
@tmathern see my question in the PR description, this is not needed now since the e2e tests can directly call the sdk right?